src/views/Pipeline.vue

```diff
@@ -277,6 +279,16 @@ export default defineComponent({
     })
   },
   methods: {
+    async copyToClipboard(job: any) {
```

Comment:
Either pass Job information as string or improve name. We could use same method at different places, if we make passing job compulsory we wouldn't be able to reuse.

Reply:
Sir, I updated the name of the function, because we want to pass whole as param instead of jobInformation as a string.